doc/contribute/index.rst

```diff
@@ -761,3 +761,31 @@ Code component README template
 ==============================
 
 .. literalinclude:: code_component_README
+
+
+Contribution Roles and Responsibilities
+***************************************
+
+Zephyr community has a number of working groups and community members that
```

Comment:
We can remove the duplicated material here (now that you've added it to
dev_env_and_tools.rst) and simplify this section with this rewrite:

```
The Zephyr project defines a development process workflow using
GitHub **Issues** to track feature, enhancement, and bug reports
together with GitHub **Pull Requests** (PRs) for submitting and
reviewing changes.  Zephyr community members work together
with `project working groups
<https://wiki.zephyrproject.org/Zephyr-Committee-and-Working-Group-Meetings>`_
to review these Issues and PRs, managing feature enhancements and
quality improvements of Zephyr through its regular releases, as outlined
in this `program management overview
<https://wiki.zephyrproject.org/Program-Management>`_.

We can only manage the volume of Issues and PRs, by requiring
timely reviews, feedback, and responses from the reviewing working groups,
area maintainers, and contributors, both for initial submissions
and for followup questions and clarifications.  Read about the
project's :ref:`development processes and tools
<dev-environment-and-tools>` and specifics about 
:ref:`review timelines <review_time>` to learn about the project's
goals and guidelines for our active developer community.
```

Reply:
agreed